libcxx/vendor/llvm/default_assertion_handler.in

```diff
@@ -0,0 +1,23 @@
+// -*- C++ -*-
```

Comment:
Do we want to prefix this filename with `__`?

Reply:
We need to, if it gets installed into `$PREFIX/usr/include/c++/v1`.

Reply:
Sorry, I mean this specific file. During installation, it gets renamed to `__assertion_handler`, so the generated include directory will contain the prefixed version.

Reply:
I guess I would leave it without `__`, since that makes it more obvious that the two files are named differently.